src/preview/assets/js/breadcrumb.js

```diff
@@ -0,0 +1,80 @@
+// import anime from 'animejs';
+import ecr from 'element-client-rect';
+
+export default class Breadcrumb {
+  constructor(el) {
+    this.el = el;
+
+    this.init();
+    this.initListeners();
+
+    this.setInitialState();
+  }
+
+  init() {
+    this.options = {};
+
+    this.classes = {
+      dom: {
+        trigger: 'pew_breadcrumb__trigger',
```

Comment:
Is trigger really a good name. Everything can be a trigger, and by the name you don't know what it does trigger